testsetups/keeper.go

```diff
@@ -94,8 +94,6 @@ func (k *KeeperBlockTimeTest) Setup() {
 		Expect(err).ShouldNot(HaveOccurred(), "Deploying KeeperConsumerPerformance instance %d shouldn't fail", i+1)
 		k.keeperConsumerContracts = append(k.keeperConsumerContracts, keeperConsumerInstance)
 		Expect(err).ShouldNot(HaveOccurred())
-		// err = k.linkTokenContract.Transfer(keeperConsumerInstance.Address(), big.NewInt(1e18))
```

Comment:
Do we still need these commented out lines?

Reply:
Nope, they were an unnecessary step that I took out as an experiment to see if they were truly unnecessary. I had forgotten to fully remove them, doing so now.